ext/standard/basic_functions.stub.php

```diff
@@ -1125,7 +1125,7 @@ function gettimeofday(bool $as_float = false): array|float {}
 #endif
 
 #ifdef HAVE_GETRUSAGE
-function getrusage(bool $mode = 0): array|false {}
+function getrusage(bool $mode = false): array|false {}
```

Comment:
Huh, this signature is not right. This should be `int $mode = 0`. It could be converted to accept a bool, but I'm not sure it's a good idea from a forward-compatibility perspective, as the underlying getrusage API actually has three modes.

Reply:
Ah, I only checked that its type was changed to `bool`, and unfortunately I didn't look up the implementation.